tests/Microsoft.DotNet.Framework.Docker.Tests/ImageTests.cs

```diff
@@ -93,6 +93,39 @@ public void VerifyImages(ImageDescriptor imageDescriptor)
             }
         }
 
+        [Theory]
+        [MemberData(nameof(GetVerifyImagesData))]
+        public void VerifyImagesWithWebApps(ImageDescriptor imageDescriptor)
```

Comment:
Seems to be a lot of duplication with the other test case.  Consider refactoring to encapsulate the commonality in a single location.